.github/workflows/test.yml

```diff
@@ -27,11 +27,11 @@ jobs:
 
   # separate job to set as required in branch protection,
   # as the above change each time the Node versions change
-  test:
+  validate-types:
     runs-on: ubuntu-latest
-    needs: test_matrix
+    needs: test
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v2
       - run: npm ci
-      - run: "npm run validate:ts"
\ No newline at end of file
+      - run: "npm run validate:ts"
```

Comment:
the "test" job name is significant, it is part of the branch protection for this repository. The main reason why I split up tests into two jobs, is that the names for the node version test matrix change each time we change the versions we want to test:

<img width="924" alt="image" src="https://user-images.githubusercontent.com/39992/111014120-2de04d00-8357-11eb-90ba-78fc74fc2a43.png">

while the final "test" job name remains the same.

<img width="955" alt="image" src="https://user-images.githubusercontent.com/39992/111014112-2325b800-8357-11eb-83e5-2f44be1aa15e.png">

All this to change, the job names are named that way for the sake of simpler branch protection settings

Reply:
Created a [post](https://github.community/t/if-a-check-run-name-changes-it-keeps-being-required/167669?u=oscard0m) in GitHub Community to see if we can come up with a better solution. This looks a bit hacky to me but I guess there is no choice for the moment.